controls/package_spec.rb

```diff
@@ -74,13 +74,14 @@
   describe package(val_syslog_pkg) do
```

Comment:
@atomic111 @artem-sidorenko Should we accept systemd journal as a default solution here too?

Reply:
@chris-rock I think yes